src/js/components/DropButton/DropButton.js

```diff
@@ -10,6 +10,7 @@ const DropButton = forwardRef(
   (
     {
       a11yTitle = 'Open Drop',
+      'aria-label': ariaLabel,
```

Comment:
I think the `aria-label` will work transparently via the `...rest` spread. So I don't think we need explicit attention for that.